src/lib/include/xtgeo/geometry.hpp

```diff
@@ -122,13 +133,22 @@ init(py::module &m)
                    "False otherwise.");
     m_geometry.def("is_xy_point_in_quadrilateral", &is_xy_point_in_quadrilateral,
                    "Return True if a XY point is inside a quadrilateral seen from , "
-                   "above. False otherwise.");
+                   "above. False otherwise.",
+                   py::arg("x"), py::arg("y"), py::arg("p1"), py::arg("p2"),
+                   py::arg("p3"), py::arg("p4"),
+                   py::arg("tolerance") = numerics::TOLERANCE);
```

Comment:
If the function is exposed with python, and have a key with default, then need to add explictly to py::arg(...)